docs/concepts/Security-Best-Practices.md

```diff
@@ -3,62 +3,96 @@ title: Best practices for a secure software supply chain
 description: Best practices for securing your software supply chain using NuGet & GitHub.
 author: JonDouglas
 ms.author: jodou
-ms.date: 02/08/2021
```

Comment:
see: https://github.com/NuGet/docs.microsoft.com-nuget/pull/3341#issuecomment-2364776821

it says:

> If you don't specify ms.date, the date will be updated automatically.